### ISSUE TITLE
Sidebar width is so big

### ISSUE THREAD

**Issue by mohamedhagag** [Openworx/backend_theme#47]
https://github.com/Openworx/backend_theme/blob/700018ca043565848d19c9038996990c1a012bf1/backend_theme_v10/static/src/less/sidebar.less#L6

Dears, 

Why 180px ? 

I believe it's enough to show the icons only. This will save space and in the same time will ease the navigation.

Also hints will show the application name, plz have a look at the attached screenshot below - which have max-width: 70px and overflow-y: hidden for .app-sidebar-panel class:

![screenshot_2018-01-27_05-21-07](https://user-images.githubusercontent.com/1042088/35468195-11763df4-0322-11e8-89eb-050f8f9b2ad1.png)

Regards,

**Comment by mgielissen** [comment]
Do you have an example for hints?

**Comment by mohamedhagag** [comment]
yes it's in the previous screenshot - the word Accounting in the black bubble is a hint when we hover on the accounting icon (orange one).
here's another screenshot, I'm pointing to the Cyan Icon of purchase and it's showing the purchase text in the hint bubble.
![screenshot_2018-01-27_20-55-14](https://user-images.githubusercontent.com/1042088/35475303-996d6bb0-03a4-11e8-9cfc-97019527ee3a.png)

**Comment by mgielissen** [comment]
CSS code for hints?

**Comment by mohamedhagag** [comment]
hints is working out of box no need to add CSS
I can purpose a PR if you want.

**Comment by youring** [comment]
What about bigger icons and removing menu name
![image](https://user-images.githubusercontent.com/4132418/39300541-a32247c8-497e-11e8-91ba-9a8a7ce7110c.png)

**Comment by mgielissen** [comment]
Nice, It looks more like Ubuntu Unity interface. You can purpose a PR.

**Comment by SalahAdDin** [comment]
@mgielissen, a pull request was already proposed.
